src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -25,8 +25,7 @@ public class ConvertToJsonCommand : PSCmdlet
         [AllowNull]
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-linux`
ConvertTo-Json.DateTime works
```powershell
Expected strings to be the same, but they were different.
String lengths are both 108.
Strings differ at index 35.
Expected: '...004+05:00"...'
But was:  '...004+00:00"...'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1: line 174
174:         $date | ConvertTo-Json -Compress | Should -BeExactly $expected
```